src/bosh-director/lib/bosh/director/deployment_plan/dynamic_network.rb

```diff
@@ -21,16 +22,24 @@ def self.parse(network_spec, availability_zones, logger)
           subnets = network_spec['subnets'].map do |subnet_properties|
             name_servers = name_server_parser.parse(subnet_properties['name'], subnet_properties)
             cloud_properties = safe_property(subnet_properties, 'cloud_properties', class: Hash, default: {})
+            prefix = safe_property(subnet_properties, 'prefix', class: Integer, default: nil)
+            if prefix.nil?
+              prefix = 32 # we need to set the ipv4 default value (dynamic networks only support ipv4)
```

Comment:
It seems odd to be setting `prefix = 32` here `if prefix.nil?` when we are defaulting it to `nil` above on line 25...

Should we be setting `default: 32` and then using the following guard:

```ruby
raise NetworkInvalidProperty, "Prefix property is not supported for dynamic networks." unless prefix == 32
```

Reply:
Could we use a constant like: `<SomeRubyClass::>IPV4_PREFIX_SIZE = 32` here